test/metabase/api/search_test.clj

```diff
@@ -215,16 +215,16 @@
                (search-request-data :crowberto :q "test collection"))))))
   (testing "It limits matches properly"
     (with-search-items-in-root-collection "test"
-      (is (= 2 (count (search-request-data :crowberto :q "test" :limit "2" :offset "0"))))))
+      (is (>= 2 (count (search-request-data :crowberto :q "test" :limit "2" :offset "0"))))))
   (testing "It offsets matches properly"
     (with-search-items-in-root-collection "test"
       (is (<= 4 (count (search-request-data :crowberto :q "test" :limit "100" :offset "2"))))))
   (testing "It offsets without limit properly"
     (with-search-items-in-root-collection "test"
-      (is (= 5 (count (search-request-data :crowberto :q "test" :offset "2"))))))
+      (is (<= 5 (count (search-request-data :crowberto :q "test" :offset "2"))))))
   (testing "It limits without offset properly"
     (with-search-items-in-root-collection "test"
-      (is (= 2 (count (search-request-data :crowberto :q "test" :limit "2"))))))
+      (is (>= 2 (count (search-request-data :crowberto :q "test" :limit "2"))))))
```

Comment:
It's not clear to me why we sometimes relax the lower bound, sometimes the upper bound.

Reply:
It's related to the change in search ordering.

There an updated search ordering, and `search-request-data` removes databases (for a good reason -- e.g. if you have test databases they will show up in the search results, so the tests would be unstable based on the testing env).

So that is why the results here changed -- I think I can use `=` instead tho, after all.